ansible/requirements.yml

```diff
@@ -12,10 +12,11 @@
 - src: stackhpc.beegfs
   version: v19.8.2
 - src: stackhpc.monasca-agent
+  version: 1.0.4
```

Comment:
Latest on ansible galaxy is 2.0.0 but github seems to be up to 2.10.1. Not sure if we need to bump this more?

Reply:
2.*.* assumes use of python 3, hence contraining the version to 1.0.4.